packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
   - package: dbt-labs/dbt_utils
-    version: 0.9.2
\ No newline at end of file
+    version: 0.9.6
```

Comment:
Do we really need to change this version?

Reply:
The check failed so I had to change but its  not mandatory